generated/nidaqmx/nidaqmx.proto

```diff
@@ -1776,6 +1761,21 @@ enum DeviceUInt32ArrayAttribute {
   DEVICE_ATTRIBUTE_ACCESSORY_SERIAL_NUMS = 12143;
 }
 
+enum DeviceInt32ArrayAttribute {
+  DEVICE_INT32_ARRAY_ATTRIBUTE_UNSPECIFIED = 0;
+  DEVICE_ATTRIBUTE_AI_SUPPORTED_MEAS_TYPES = 12242;
+  DEVICE_ATTRIBUTE_AO_SUPPORTED_OUTPUT_TYPES = 12243;
+  DEVICE_ATTRIBUTE_CI_SUPPORTED_MEAS_TYPES = 12244;
+  DEVICE_ATTRIBUTE_CO_SUPPORTED_OUTPUT_TYPES = 12245;
+  DEVICE_ATTRIBUTE_AI_SAMP_MODES = 12252;
+  DEVICE_ATTRIBUTE_AO_SAMP_MODES = 12253;
+  DEVICE_ATTRIBUTE_CI_SAMP_MODES = 12254;
+  DEVICE_ATTRIBUTE_CO_SAMP_MODES = 12255;
+  DEVICE_ATTRIBUTE_NAV_SUPPORTED_MEAS_TYPES = 12323;
+  DEVICE_ATTRIBUTE_NAV_TRIG_USAGE = 12324;
+  DEVICE_ATTRIBUTE_AI_DIG_FLTR_TYPES = 12551;
+}
+
 enum ExportSignalDoubleAttribute {
   EXPORTSIGNAL_DOUBLE_ATTRIBUTE_UNSPECIFIED = 0;
```

Comment:
FYI, it's kind of weird that this shuffled the order of the enum blocks, but I think I see why. `get_attributes_split_by_sub_group` processes the attributes in `attributes.py` order and sorts them into dictionaries. The order of data types in `categorized_attributes` depends on the position of the first attribute with that data type within `attributes.py`.